### PR TITLE
Bump Serilog to 3.1.1

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/Destructurama.JsonNet.Tests/Approval/Destructurama.JsonNet.approved.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace Destructurama
 {
-    public static class LoggerConfigurationAppSettingsExtensions
+    public static class LoggerConfigurationJsonNetExtensions
```
Comment (sungam3r) [inline src/Destructurama.JsonNet.Tests/Approval/Destructurama.JsonNet.approved.txt:3]: @nblumhardt  I think this class was initially copied from other project. I rename it in light of major version bump. OK?

File: src/Destructurama.JsonNet/Destructurama.JsonNet.csproj [hunk 2]
```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```
Comment (sungam3r) [inline src/Destructurama.JsonNet/Destructurama.JsonNet.csproj:4]: @nblumhardt Keep net462 or drop?

Comment (nblumhardt) [reply src/Destructurama.JsonNet/Destructurama.JsonNet.csproj:4]: I think there's value in 462, I'd also consider adding net6.0; no strong feelings though

Comment (sungam3r) [reply src/Destructurama.JsonNet/Destructurama.JsonNet.csproj:4]: I'll publish `net462;netstandard2.0` in v3. We can add net6 later if some issue arises.

**Comment by sungam3r** [comment]
https://github.com/destructurama/json-net/releases/tag/3.0.0
https://www.nuget.org/packages/Destructurama.JsonNet/3.0.0

**Pull request merged**